modules/rosa-quick-start-install-configure-cli-tools.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * rosa_quick_start/rosa-quick-start.adoc
+
+[id="rosa-quick-start-install-configure-cli-tools_{context}"]
+= Installing and configuring the required CLI tools
+
+Use the following steps to install and configure the AWS, {product-title} (ROSA) and OpenShift CLI tools on your workstation.
+
+.Prerequisites
+
+* You created an AWS account.
+* You created a Red Hat account.
++
+[NOTE]
+====
+You can create a Red Hat account by navigating to link:https://cloud.redhat.com[cloud.redhat.com] and selecting *Create an account*.
```

Comment:
can we update the link to console.redhat.com?

Reply:
Great idea! I have now resolved this.